test_scripts/Polices/External_UCS/External_Consent_ON/012_ATF_Policies_External_Consent_ON_through_ignition_cycles_user_disallowed_RPCs.lua

```diff
@@ -0,0 +1,170 @@
+-------------------------------------- Requirement summary -------------------------------------------
+-- [Policies] External UCS: "ON" status between ignition cycles
+-- [Policies] External UCS: "ON" - user_disallowed RPCs
+--
+------------------------------------------------------------------------------------------------------
+------------------------------------ General Settings for Configuration ------------------------------
+------------------------------------------------------------------------------------------------------
+require('user_modules/all_common_modules')
+local common_functions_external_consent = require('user_modules/shared_testcases_custom/ATF_Policies_External_Consent_common_functions')
+------------------------------------------------------------------------------------------------------
+---------------------------------------Preconditions--------------------------------------------------
+------------------------------------------------------------------------------------------------------
+-- Start SDL and register application
+common_functions_external_consent:PreconditonSteps("mobileConnection","mobileSession")
+-- Activate application
+common_steps:ActivateApplication("Activate_Application_1", config.application1.registerAppInterfaceParams.appName)
+------------------------------------------------------------------------------------------------------
+------------------------------------------- Functions ------------------------------------------------
+------------------------------------------------------------------------------------------------------
+local function CheckGroup001IsConsented()
```

Comment:
@dboltovskyi, may the function should have different name, because it checks USER_DISALLOWED.